Makefile

```
@@ -387,6 +387,7 @@ endif
 
 argoexec-image: ## Build the executor image
 argoexec-nonroot-image:
+argo-workflows-crdinstaller-image: ## Build the CRD installer image
```

checkmake (0.3.2)

[warning] Target "argo-workflows-crdinstaller-image" should be declared PHONY.

(phonydeclared)